modules/react/_examples/stories/examples/AriaLiveRegions/VisibleLiveRegion.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {AriaLiveRegion} from '@workday/canvas-kit-react/common';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
+import {FormField} from '@workday/canvas-kit-preview-react/form-field';
+import {Flex} from '@workday/canvas-kit-react/layout';
+import {Text} from '@workday/canvas-kit-react/text';
+import {TextInput} from '@workday/canvas-kit-react/text-input';
+import {system, base} from '@workday/canvas-tokens-web';
+import {createStyles, px2rem} from '@workday/canvas-kit-styling';
+
+const liveRegionStyle = createStyles({
+  border: `${px2rem(1)} solid ${base.cantaloupe400}`,
+  backgroundColor: base.cantaloupe100,
+  padding: system.space.x4,
+  display: 'block',
+  margin: system.space.x4 + ' 0',
+  width: '28rem',
```

Comment:
Quick question, what is the `margin` calculation?

```suggestion
  border: `${px2rem(1)} solid ${system.color.border.caution.default}`,
  backgroundColor: system.color.bg.caution.default,
  padding: system.space.x4,
  display: 'block',
  margin: system.space.x4 + ' 0',
  width: calc.multiply(system.space.x16, 7),
```